frontend/packages/knative-plugin/src/utils/create-eventsources-utils.ts

```diff
@@ -77,6 +100,27 @@ export const getEventSourceData = (source: string) => {
         },
       },
     },
+    kafkasource: {
+      spec: {
```

Comment:
can we have `resources` and `requests` as well under this spec with default initial values instead of having under `limits` as these values are used only in Kafka Source so am not sure if want to have it separate and with this we can avoid https://github.com/openshift/console/pull/4899/files#diff-745c0439e378574e2e12727d3caa5522R41-R61

WDYT?

Reply:
If you see the way ResourceLimitSections is written it requires limit value, unit, default unit etc so we need to have all that info in the form itself. It made sense to keep it outside and then modify it according to our needs in a similar way we do in other import forms.